CtaReactiveMaster/Sources/APIClient/NewsAPIRequest.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  NewsAPIRequest.swift
+//  CtaReactiveMaster
+//
+//  Created by 化田晃平 on R 3/01/15.
+//
+
+import Foundation
+
+//NewsAPIのエンドポイントのパス
+enum Endpoint{
```

Comment:
```suggestion
enum EndPoint {
```
かな？

どちらでも良いけど下のメソッドのケースと合わせたい

Reply:
了解です！